### DISCUSSION TITLE
JSON-RPC error: RPC error response:

### DISCUSSION THREAD

**Discussion by ghost** [ordinals/ord#1750] [Q&A]
Hi, please share ideas about my mistake. 
Thank you.

Win 11, ORD 0.5.0, full node.
bitcoin.config (txindex=1 datadir=D: \ rest=1 server=1 rpcconnect=127.0.0.1).
Indexing ORD finished fine, bitcoind and ORD launched.
Now (this error did not occur before), after instruction
ord --cookie-file "D:\.cookie" wallet balance

I get an error ORD
JSON-RPC error: RPC error response: RpcError { code: -5, message: "No such mempool or blockchain transaction. Use gettransaction for wallet transactions.", data: None }
And get a record Bitcoind
ERROR: FindTx: Deserialize or I/O error - ReadCompactSize(): size too large: iostream error

Other instructions work correctly (wallet create/receive etc.)
All Bitcoin Core and ORD files on drive D.
P.s index.redb ORD file on drive C.

**Comment by belovie** [comment]
I'm having the same issue

**Comment by tuncatunc** [comment]
I'm having a similar issue here in testnet.
```
bitcoind --version
Bitcoin Core version v24.0.1
```

```
./ord --bitcoin-rpc-pass <password>  --bitcoin-rpc-user <user> --testnet info

[indexing blocks] ████████████████████████████████████████████████████████████████████████████████████████████████████████████████████░ 2413211/2431877
[2023-05-05T12:41:08Z ERROR ord::index::updater] Couldn't receive txs Failed to fetch raw transaction: code -5 message No such mempool transaction. Use -txindex or provide a block hash to enable blockchain transaction queries. Use gettransaction for wallet transactions.
error: failed to get transaction for 9c475139490215608ebb8e153b02f9794029db69fa15f0f65a5957037f4cba2d
```

**Comment by ronyevernaes** [comment]
I have this issue on MacOS

**Comment by sneurlax** [comment]
Y'all may need to relaunch your node with the `-reindex` flag

You have txindex=1 set, but if your node was launched before/without this setting, it will need to reindex.

**Reply by angree** [reply]
this is not true. I just re-indexed the whole thing. Took like 4 days on my ssd and STILL same exact error for same exact transaction.. Insane. I've been downloading and re-indexing this for almost two weeks.

**Reply by sneurlax** [reply]
Sorry for wasting your time in that case!  It fixed it for me but that may not apply to everyone.

**Reply by atproto** [reply]
Such a poor answer, without any kind of a warning! Now I have to waste my few days to rebuild the blocks. I was too quick to run this and missed @angree 's valuable comment.

**Comment by alpha128bitz** [comment]
can anyone solve it? The process progresses with constant errors. After a point it becomes truly unbearable.

**Comment by Detoo** [comment]
Setting JSON-RPC url for bitcoin core with explicit port (`--rpc-url=127.0.0.1:8332`) fixed it for me
